README

```diff
@@ -1,14 +0,0 @@
-This is the distribution of the Daikon invariant detector,
```

Comment:
Why has the README file been removed?

If you propose to do so, then I would prefer such changes to appear in a separate pull request, not mixed in with the exception-handling feature.